### DISCUSSION TITLE
Stress definition in coupled HM analyses of unsaturated soil

### DISCUSSION THREAD

**Discussion by nlosacco** [idaholab/moose#17609] [Q&A Modules: General] [answered]
Hi all,

I am willing to start using MOOSE for my research in geotechnical engineering (mainly for slope stability and tunnelling problems) which involve coupled hydro-mechanical analyses in partly-saturated soils.
I looked at the documentation but I wasn't able to understand which (effective) stress definition is used for the mechanical part of unsaturated porous materials.
The porous flow documentation:
[https://mooseframework.inl.gov/modules/porous_flow/governing_equations.html](url)
states that, for the solid mechanics bit:
"[...] for multi-phase situations P_f = [weighed sum of pore pressures for each phase] is also used. Yet, other expressions involve Bishop's parameter".
The sentence above is a bit cryptic: is Bishop's stress used (i.e. sigma_tot - P_gas * delta + chi * (P_gas - P_water)*delta
with chi the Bishop's parameter)? Can the two independent stress variables approach be used (e.g. sigma_tot - P_water and P_gas - P_water)?
Where can I find some more detailed explanation about how MOOSE can deal with this? Can you point me to any examples on the topic?
Thank you in advance for any help,
N

**Comment by GiudGiud** [comment]
@WilkAndy

**Comment by WilkAndy** [comment, marked as answer]
The documentation is poorly worded, my apologies.  I will fix it to read something like "[...] for multi-phase situations P_f = [weighed sum of pore pressures for each phase] is also used. These forms are coded into the PorousFlowEffectiveFluidPressure Material.  Other expressions, such as one involving Bishop's parameter, could be included into PorousFlow if users request."

I'll fix the documentation in the coming days.

**Reply by nlosacco** [reply]
@WilkAndy Thank you very much for your clarification. I think the possibility to use Bishop's stress would be a valuable addition to the program for those interested in solving partly saturated soil mechanics problems like, e.g. , applications involving stability of slopes.
Best,
N

**Reply by WilkAndy** [reply]
You can submit an "feature enhancement" issue if you like, specifying exactly what you need (eg, is Bishop's param an AuxVariable, should this work for N phases, what about M components, etc).

a
